examples/stable-video-diffusion/README.md

```diff
@@ -0,0 +1,37 @@
+# Stable Video Diffusion
+A nos integration example of [Stable Video Diffusion(SVD)](https://github.com/Stability-AI/generative-models). The SVD is a diffusion model trained for short video generation task, it uses a still image for conditioning to creates a short video clip. 
+The orignal paper can be found [here](https://stability.ai/research/stable-video-diffusion-scaling-latent-video-diffusion-models-to-large-datasets).
+
+## Deployment
+Simply run the following command to serve up the model:
+``` bash
+nos serve up -c serve.yaml
+```
+
+## Inference
+Here is an example how to run the inference:
+``` python
+from PIL import Image
+from nos.client import Client
+
+client = Client("[::]:50051")
+model = client.Module("stable-video-diffusion")
+
+image_link = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/svd/rocket.png?download=true"
+
+response: Iterable[Image.Image] = model.image2video(image=image_link, _stream=True)
```

Comment:
Do a `image: Image.Image = request.get(...)` here and pass the image instead to the `image2video` method.

Reply:
Got it, changed.